tests/python/gaia-ui-tests/gaiatest/visual_components/visual_component.py

```diff
@@ -0,0 +1,7 @@
+from gaiatest.apps.base import PageRegion
+
+
+class VisualComponent(PageRegion):
```

Comment:
Although I have to say, I don't really know a better name. Perhaps FormControl or something?